File: src/main/java/com/linebot/model/qiita/ItemSummary.java
package com.linebot.model.qiita;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class ItemSummary {
    private String id;
    private String title;
    private String url;
    private int likesCount;
    private int pageViewsCount;
    private int stockersCount;
}

File: src/main/kotlin/com/linebot/model/qiita/ItemSummary.kt
package com.linebot.model.qiita

data class ItemSummary(
        var id: String? = null,
        var title: String? = null,
        var url: String? = null,
        var likesCount: Int = 0,
        var pageViewsCount: Int = 0,
        var stockersCount: Int = 0
)
